lib/command/Command.js

```diff
@@ -369,9 +369,9 @@ class Command {
                         args = response.args || args;
                     }
                 }
-                reply = typeof this.invalidUsageMessage === "function" ? this.invalidUsageMessage(msg) : this.invalidUsageMessage;
+                reply = typeof this.invalidUsageMessage === "function" ? this.invalidUsageMessage(msg) : this.invalidUsageMessage.replace(/%prefix%/g, msg.prefix).replace(/%label%/g, this.fullLabel);
                 if(reply) {
-                    msg.channel.createMessage(reply.replace(/%prefix%/g, msg.prefix).replace(/%label%/g, this.fullLabel));
+                    msg.channel.createMessage(reply);
```

Comment:
We should probably add support for replacing templates in the embed in relevant locations